--- djview4-4.5/desktopfiles/Makefile.in.orig	2009-02-19 17:51:51.000000000 +0100
+++ djview4-4.5/desktopfiles/Makefile.in	2010-02-12 08:47:57.000000000 +0100
@@ -38,7 +38,8 @@
 RM = @RM@
 
 datadir_djview = ${datadir}/djvu/djview4/desktop
-
+desktopdir = /usr/share/applications
+iconsdir = /usr/share/icons
 
 # -------- all
 
@@ -68,11 +69,9 @@
 install: install-djview-files 
 
 install-djview-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djview}
-	${INSTALL_PROGRAM} register-djview-menu ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} *djview4.png ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} *djview4.svgz ${DESTDIR}${datadir_djview}
-	${INSTALL_PROGRAM} ${srcdir}/*djview4.desktop ${DESTDIR}${datadir_djview}
+	${INSTALL} -d ${DESTDIR}${desktopdir} ${DESTDIR}${iconsdir}/hicolor/32x32/apps
+	${INSTALL_DATA} ${srcdir}/hi32-djview4.png ${DESTDIR}${iconsdir}/hicolor/32x32/apps/djvulibre-djview4.png
+	${INSTALL_DATA} ${srcdir}/djvulibre-djview4.desktop ${DESTDIR}${desktopdir}/djvulibre-djview4.desktop
 	-@echo "Run ${datadir_djview}/register-djview-menu install"
 	-@echo " to register the djview3 menu entries."
 
