--- djview4-4.0/desktopfiles/Makefile.in.orig	2007-01-17 03:30:46.000000000 +0100
+++ djview4-4.0/desktopfiles/Makefile.in	2007-02-04 23:48:41.435031448 +0100
@@ -35,7 +35,8 @@
 RM = @RM@
 
 datadir_djview = ${datadir}/djvu/djview4/desktop
-
+desktopdir = /usr/share/applications
+iconsdir = /usr/share/icons
 
 # -------- all
 
@@ -55,10 +56,9 @@
 install: install-djview-files 
 
 install-djview-files: FORCE
-	${INSTALL} -d ${DESTDIR}${datadir_djview}
-	${INSTALL_PROGRAM} register-djview-menu ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview4.png ${DESTDIR}${datadir_djview}
-	${INSTALL_DATA} ${srcdir}/*djview4.desktop ${DESTDIR}${datadir_djview}
+	${INSTALL} -d ${DESTDIR}${desktopdir} ${DESTDIR}${iconsdir}/hicolor/32x32/apps
+	${INSTALL_DATA} ${srcdir}/hi32-djview4.png ${DESTDIR}${iconsdir}/hicolor/32x32/apps/djvulibre-djview4.png
+	${INSTALL_DATA} ${srcdir}/djvulibre-djview4.desktop ${DESTDIR}${desktopdir}/djvulibre-djview4.desktop
 	-@echo "Run ${datadir_djview}/register-djview-menu install"
 	-@echo " to register the djview3 menu entries."
 
--- djview4-4.2/desktopfiles/djvulibre-djview4.desktop.orig	2007-08-01 05:31:12.000000000 +0200
+++ djview4-4.2/desktopfiles/djvulibre-djview4.desktop	2008-01-12 15:30:13.292034239 +0100
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Type=Application
-Categories=Qt;Graphics
+Categories=Qt;Graphics;Viewer;
 MimeType=image/x-djvu;image/x.djvu;image/vnd.djvu;
 Name=DjView4
 Exec=djview4 %f
